### ISSUE TITLE
[Bug]: M5StickCPlus Listener LED on/off Backwards

### ISSUE THREAD

**Issue by commputethis** [josephdadams/TallyArbiter#474]
### What happened?

After a lot of testing with the M5StickCPlus Listener and the M5Stack example for blinking LED, I found that LOW is on and HIGH is off, which is backwards from the M5StickC.

This code starts on line 618 of the m5stickc-listener ino file.
```
    #if TALLY_EXTRA_OUTPUT
    if (actualType == "\"program\"") {
      digitalWrite (led_program, LOW);
      digitalWrite (led_preview, HIGH);
      digitalWrite (led_aux, HIGH);
    } else if (actualType == "\"preview\"") {
      digitalWrite (led_program, HIGH);
      digitalWrite (led_preview, LOW);
      digitalWrite (led_aux, HIGH);
    } else if (actualType == "\"aux\"") {
      digitalWrite (led_program, HIGH);
      digitalWrite (led_preview, HIGH);
      digitalWrite (led_aux, LOW);
    } else {
      digitalWrite (led_program, HIGH);
      digitalWrite (led_preview, HIGH);
      digitalWrite (led_aux, HIGH);
    }
    #endif
```

You will also notice I added an escape character and extra quotes on the actualType as that was required.  I am not sure if it is something to do with running TallyArbiter in test mode or not, but I did have to do that to get it to work.  Also, the "program" actualType if statement had "preview" in it.  So there were two preview if statements.

### Version

3.0.4

### Distribution

Docker

### OS

Linux

### If applicable, What Listener Clients are You Using?

M5 Stick

**Comment by josephdadams** [comment]
Can you submit a PR for this?

**Comment by commputethis** [comment]
yeah. I thought about that, but I will want to test it in my production environment first, vs just at home with the built in test mode, to verify the \" are needed outside of test mode.  Also, I will need to test with a M5StickC (not Plus) to verify it is still HIGH for on and LOW for off.  If it is, then we'll need to add separate lines of code to accommodate the difference.  I'm thinking maybe use variables for led_on and led_off with different values for non-plus and plus.  Then not need to have separate sections throughout the code for it.  

It might take me a week or so to get to it.  I don't live close to the church and will be away this weekend.  I'll get it flushed out and let you know how it goes.

**Comment by commputethis** [comment]
I was able to verify with both the M5StickC and M5StickCPlus that having the LOW and HIGH swapped works.  I will still need to test this at church where I have an ATEM source to test whether "program" should be "\"program\"" or  just "program" outside of using the built in test source.

I tried looking through the source code to see how the test system is sending the program, preview, aux, but I wasn't able to figure it out.
